JavaScript/Angular.md

```diff
@@ -969,134 +969,134 @@ class MyComponent implements OnInit, OnChanges {
 }
 ```
 
-## Dependency Injection
+## Dependency injection
 
 DI is utilized heavily in Angular, primarily for providing instances of either primitive values or class instances.
 
-DI is a very complex topic and warrants a separate discussion which would be out of scope for this Handbook. Make sure to check out the [official DI guide](https://angular.io/guide/dependency-injection) which explains many details. You can also have a look at [this repo](https://github.com/fvoska/angular-di-demo) for some examples.
+DI is a very complex topic and requires a separate discussion which would be out of the scope of this handbook. Make sure to check out the [official DI guide](https://angular.io/guide/dependency-injection) which explains many details. You can also have a look at [this repo](https://github.com/fvoska/angular-di-demo) for some examples.
 
 As of Angular 6, make sure to use `{ providedIn: 'root' }` [for singletons](https://angular.io/guide/singleton-services#providing-a-singleton-service) whenever possible.
 
-DI is very useful for testing purposes, as shown in next chapter.
+DI is very useful for testing purposes, as shown in the next chapter.
```

Comment:
I actually just noticed that I reordered some chapters after writing this line, so the chapter about testing is no longer immediately after DI chapter.

```suggestion
DI is very useful for testing purposes, as shown in a later chapter.
```

Reply:
Using "in a later chapter" is OK.